apps/src/sharedJsxStyles.js

```diff
@@ -6,10 +6,6 @@
 
 module.exports = {
   colors: {
-    charcoal: '#5b6770',
-    green: '#b9bf15',
-    white: '#fff',
-    orange: '#ffa400',
```

Comment:
Yeah, maybe; I went conservative not moving anything out of apps, but since there's nothing but colors in this file right now maybe that's the right thing to do.  I was going to say other common styles / style bundles might eventually live here, but personally I prefer doing that case-by-case anyway (e.g. [ToggleButtonStyles.js](https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/templates/ToggleButtonStyles.js)).  I'll tear this out before I merge.